Neos.Flow/Classes/Core/LockManager.php

```diff
@@ -135,7 +135,9 @@ public function unlockSite()
             fclose($this->lockResource);
             unlink($this->lockPathAndFilename);
         }
-        @unlink($this->lockFlagPathAndFilename);
+        if (file_exists($this->lockFlagPathAndFilename)) {
+            @unlink($this->lockFlagPathAndFilename);
+        }
```

Comment:
The shutup operator seems to be ignored for PHP 8 - at least I got a Warning that the file didn't exist

Reply:
Yep, see https://github.com/neos/flow-development-collection/pull/2287/checks?check_run_id=1974309752#step:12:28
PHP 8 will behave differently and I guess we need to try/catch here (our errorHandler will convert the error to an Exception) or introduce the "double shrug 🤷‍♀️🤷‍♂️()" function.
https://php.watch/versions/8.0/fatal-error-suppression